taas/config.d/organizations.yml

```diff
@@ -4,30 +4,67 @@ sources:
         options:
             build_by_default: False
         beta-v1:
-            url: https://docs.google.com/spreadsheets/d/1oa7Wknz64dMQlV2XsW0ssjh6Gw5SnHVfGxDSw_5eKOQ/edit#gid=518938746
+            url: https://docs.google.com/spreadsheets/d/1GIA3FrRM1GZS3FoUDNpw35MhC_WLTdKBiiU2tUlWS84/edit#gid=0
             mapping:
-                id: HR info ID
-                label: Preferred Term
-                acronym:
-                    type: map
-                    field: ACRONYM
+                id: ID
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+                rw_id:
+                    field: RW ID
                     optional: True
-                homepage:
-                    type: map
-                    field: homepage
+                ocha_gms_id:
+                    field: OCHA_GMS ID
                     optional: True
-                fts_id:
-                    type: map
-                    field: FinancialTrackingService ID
+                label.default: Preferred Term
+                label.humanitarianresponse:
+                    field:
+                        - HRinfo Alt Term
+                        - Preferred Term
+                label.gms:
+                    field:
+                        - GMS Alt Term
+                        - Preferred Term
+                label.reliefweb:
+                    field:
+                        - RW Alt Term
+                        - Preferred Term
+                label.fts:
+                    field:
+                        - FTS Alt Term
+                        - Preferred Term
+                used_for:
+                    field: Used For
                     optional: True
-
-                # This gives us a map with a machine-friendly URL,
-                # and a human-friendly ID.
-                type.id:
+                url:
+                    field: URL
+                    optional: True
+                acronym:
+                    field: Acronymn
+                    optional: True
+                type.default.id:
                     type: link
-                    field: OrgTypeID - OrgTypePreferredTerm
+                    field: Organization Type
                     prefix: https://www.humanitarianresponse.info/en/api/v1.0/organization_types/
-                type.label:
+                    optional: True
+                type.default.label:
+                    type: link
+                    field: Organization Type
+                    from: label
+                    optional: True
+                type.reliefweb:
+                    field: RW Type
+                    optional: True
+                based_in.country.id:
```

Comment:
Oh! I didn't consider having this wrapped in a top-level `based_in`, but having discussed with @helenrc on flowdock, this seems like the perfect ways to do things, because it makes it clear we're dealing with countries, *and* we can mark the map up with extra info later if we have it.

It feels a bit awkward that we end up with a map of nulls (rather than a single null) if there's no country, but that would require a code change, and is more stylistic than anything else.